packages/babel-standalone/README.md

```diff
@@ -16,11 +16,11 @@ It's true that using Babel through Webpack, Browserify or Gulp should be suffici
 Installation
 ============
 
-There are several ways to get a copy of babel-standalone. Pick whichever one you like:
+There are several ways to get a copy of @babel/standalone. Pick whichever one you like:
 
-- Use it via UNPKG: https://unpkg.com/babel-standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
-- Install via Bower: `bower install babel-standalone`
-- Install via NPM: `npm install --save babel-standalone`
+- Use it via UNPKG: https://unpkg.com/@babel/standalone@6/babel.min.js. This is a simple way to embed it on a webpage without having to do any other setup.
```

Comment:
We need to double check that this is the correct syntax for Unpkg URLs.

Reply:
it is - syntax is as follows: `https://unpkg.com/package_name@version/path/to/file.js`

where `package_name` might be scoped